CRM/Member/Form/Membership.php

```diff
@@ -1110,6 +1110,7 @@ public function submit() {
     $joinDate = $startDate = $endDate = NULL;
     $membershipTypes = $membership = $calcDate = array();
     $membershipType = NULL;
+    $paymentInstrumentID = $this->_paymentProcessor['object']->getPaymentInstrumentID();
```

Comment:
@eileenmcnaughton it would appear that in that $this->_paymentProcess['object'] is not set for that specific test here